### ISSUE TITLE
make Meeting Data Template and Metadata Template easier to use

### ISSUE THREAD

**Issue by alanborning** [bmlt-enabled/crouton#256]
These seem a bit hard to use. One idea for improvement is to format them better, with whitespace, indentation, and comments explaining what the different parts are doing. Crouton could store both the original and a minified version, so there would be no performance hit for doing this.

**Comment by radius314** [comment]
I agree with this concept.  The idea would be to allow usage in the editor view to store multiple templates and then be able to reference by ID.

**Comment by alanborning** [comment]
That seems like a good idea. Could do both: support multiple templates, and also format them better.

**Comment by radius314** [comment]
Yes I’d do both.  And use the tinymce editor like bread uses.

**Comment by alanborning** [comment]
I'd like to work on this, if we are not in a hurry (I won't be that fast due to learning curve).

**Comment by radius314** [comment]
Totally.  Go for it.

**Comment by radius314** [comment]
I can help you stand up your local dev environment over a meeting of about
30 mins if need be.

**Comment by alanborning** [comment]
Great! I will follow up individually.
  Alan

**Comment by tempsaint** [comment]
+1
